.github/workflows/build_and_test.yml

```diff
@@ -24,6 +24,9 @@ jobs:
       - name: Run unit tests
         run: |
           python3 -m unittest discover -s mage_ai --failfast
+      - name: Run mage integrations unit tests
+        run: |
+          cd mage_integrations && python3 -m unittest discover mage_integrations.tests
```

Comment:
Currently only adding the unit tests from the `mage_integrations/tests` directory since some of the other sources's unit tests (e.g. hubspot, facebook ads) require further investigation to resolve issues with importing the required modules.